kodev

```diff
@@ -207,9 +217,10 @@ ${SUPPORTED_TARGETS}"
             assert_ret_zero $?
             ;;
         *)
-            # As kodev-run (to run the emulator) builds by default a debug build,
-            # let's do that here too.
-            export KODEBUG=1
+            if [ -z "${KODEBUG_NO_DEFAULT+x}" ]; then # no explicite --debug / --no-debug
+                # Builds by default a debug build, as kodev-run does
```

Comment:
And since I'm policing the comments, "builds a debug build by default, like kodev-run (does)"